docs/aqua-book/changelog.md

```diff
@@ -7,6 +7,13 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 * `ENHANCING` part is incremented for every syntax addition
 * `RELEASE` is the release number, shows internal compiler changes, bugfixes that keep the language untouched
 
+### [0.8.0](https://github.com/fluencelabs/aqua/releases/tag/0.8.0) – November 30, 2022
+* Structure objects can be created right in the code ([example](https://fluence.dev/docs/aqua-book/language/types.md#structures))
+* `module` key word changed to `aqua` ([docs](https://fluence.dev/docs/aqua-book/language/header/header.md#module))
```

Comment:
```suggestion
* `module` key word was renamed to `aqua` ([docs](https://fluence.dev/docs/aqua-book/language/header/header.md#module))
```

> header.md#module
Gives 404. And shouldn't it be `header.md#aqua`?

Reply:
I think we need to discuss it.